exporters/logging/src/main/java/io/opentelemetry/exporters/logging/LoggingSpanExporter.java

```diff
@@ -50,11 +53,15 @@ public ResultCode flush() {
         resultCode = ResultCode.FAILURE;
       }
     }
-    return resultCode;
+    return Futures.immediateFuture(resultCode);
   }
 
   @Override
   public void shutdown() {
-    this.flush();
+    try {
+      this.flush().get();
+    } catch (InterruptedException | ExecutionException e) {
+      logger.log(Level.WARNING, "Span Exporter threw an Exception", e);
+    }
   }
 }
```

Comment:
Note that if any of the existing exporters throw an exception during export or flush then they are in error in accordance with the API. I've therefore assumed that they don't.

Reply:
logging is fine here, agreed.